src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import org.broadinstitute.dsde.rawls.model._
+
+trait RawlsBillingProjectComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class RawlsBillingProjectTable(tag: Tag) extends Table[RawlsBillingProjectShort](tag, "BILLING_PROJECT") {
+    def projectName = column[String]("projectName", O.PrimaryKey)
+    def cromwellAuthBucketUrl = column[String]("cromwellAuthBucketUrl")
+
+    private def fromStrings = (name: String, bucketUrl: String) => RawlsBillingProjectShort(RawlsBillingProjectName(name), bucketUrl)
+    private def toStrings = (project: RawlsBillingProjectShort) => Option(project.projectName.value, project.cromwellAuthBucketUrl.value)
+
+    def * = (projectName, cromwellAuthBucketUrl) <> (fromStrings.tupled, toStrings)
+  }
+
+  val rawlsBillingProjectShortQuery = TableQuery[RawlsBillingProjectTable]
+
+  def rawlsBillingProjectByNameQuery(name: String) = Compiled {
```

Comment:
I don't have a strong opinion on this. Maybe we could keep it as is and once we better learn about it, we can standardize across all queries.